### PR TITLE
Enhancement description of Makefile variables

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Addresses  #469.

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

File: guide/using-webots-makefiles.md [hunk 1]
```diff
@@ -60,15 +60,16 @@ controller directory is "my\_controller" then the list must contain either
...
-| --------- | --------------------------------------------------------------------- |
-| CFLAGS    | Specifies a list of flags that will be passed to the gcc/g++ compiler |
-| INCLUDE   | Specifies a list of flags that will be passed to the gcc/g++ compiler |
-| LIBRARIES | Specifies a list of flags that will be passed to the linker           |
+| Variable  | Usage                                                                                                      |
+| --------- | ---------------------------------------------------------------------------------------------------------- |
+| CFLAGS    | Specifies a list of flags that will be passed to the gcc/g++ compiler                                      |
+| LFLAGS    | Specifies a list of flags that will be passed to the linker                                                |
+| INCLUDE   | Specifies directories to be added to the list of places searched by the gcc/g++ compiler for include files |
+| LIBRARIES | Specifies directories and libraries to be added to the list of places searched by the linker               |
```
Comment (omichel) [inline guide/using-webots-makefiles.md]: I would rephrase the last one as: "Specifies library search directories and libraries passed to the linker"

Comment (stefaniapedrazzi) [reply guide/using-webots-makefiles.md]: Ok, but this too is not very clear for me.

**Comment by stefaniapedrazzi** [comment]
:+1:
